demo/Google.Pubsub.V1.Demo/DemoCommands.CreateSubscription.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+// Licensed under the Apache License Version 2.0.
+
+using System;
+
+namespace Google.Pubsub.V1.Demo
+{
+    public partial class DemoCommands
+    {
+        /// <summary>
+        /// Creates a pubsub subscription to a topic in a Google Cloud Platform project.
+        /// It is assumed that the topic exists before this is called.
+        /// </summary>
+        public static void CreateSubscription(string projectId, string subscriptionId, string topicId)
+        {
+            // This creates a client (actually an instance of SubscriberClientImpl) synchronously, using the
```

Comment:
Not for the demo - but the demo is largely to teach people about the codebase. SubscriberClientImpl is public so that people can use the concrete class for DI purposes.